### PR TITLE
Update robot

### PR DESCRIPTION
**Pull request by seabbs**
Rt and secondary model forecast update. Finalised the secondary model to have a wider prior on the proportion of test positive cases that result in death and shortened the fitting window to 3 weeks. This is now locked in for evaluation.

### CODE REVIEW

**Comment by seabbs** [comment]
Currently seeing a cumulative forecast bug with all forecasts. Any ideas?

Screenshot 2020-12-14 at 11 48 00

**Comment by seabbs** [comment]
Ahh looking here there is a new year bug in the formatting code (as EpiWeeks rest at the beginning of the year).

**Comment by seabbs** [comment]
Looks like I've fixed. @nikosbosse please check the code to see if it makes sense to you. 

# Rt forecasts

Screenshot 2020-12-14 at 12 35 36
Screenshot 2020-12-14 at 12 35 51
Screenshot 2020-12-14 at 12 36 06
Screenshot 2020-12-14 at 12 36 20

# Convolution forecasts

Here struggling to track recent surge in deaths vs observed cases. 

Screenshot 2020-12-14 at 12 37 09
Screenshot 2020-12-14 at 12 37 27

File: rt-forecast/functions/format-forecast.R [hunk 1]
```diff
@@ -49,6 +47,9 @@ format_forecast<- function(forecasts,
   # drop unnecessary columns
   forecasts_format <- forecasts_format[, !c("epiweek", "region")]
   forecasts_format <- forecasts_format[target_end_date > forecast_date]
+  forecasts_format <- forecasts_format[, horizon := 1 + as.numeric(target_end_date - min(target_end_date)) / 7]
```
Comment (nikosbosse) [inline rt-forecast/functions/format-forecast.R:50]: couldn't you just do this without the assignment operator? `forecasts_format[, horizon := 1 + as.numeric(target_end_date - min(target_end_date)) / 7]`

File: rt-forecast/update-submission.R [hunk 2]
```diff
@@ -63,6 +63,6 @@ save_forecast(case_forecast, "Germany", "-case")
 save_forecast(case_forecast, "Poland", "-case")
 save_forecast(death_forecast, "Germany")
 save_forecast(death_forecast, "Poland")
-save_forecast(death_from_cases_forecast, "Germany", "-secondary", deaths_folder)
-save_forecast(death_from_cases_forecast, "Poland", "-secondary", deaths_folder)
+save_forecast(death_from_cases_forecast, "Germany", "_secondary", deaths_folder)
```
Comment (nikosbosse) [inline rt-forecast/update-submission.R:66]: Johannes will love it :-D

**Review by nikosbosse — COMMENTED** [review]
Yes I think that looks perfectly reasonable

File: rt-forecast/functions/format-forecast.R (hunk 1 above)
Comment (seabbs) [reply rt-forecast/functions/format-forecast.R:50]: yes, I like it as a visual reminder of what is happening.

File: rt-forecast/update-submission.R (hunk 2 above)
Comment (seabbs) [reply rt-forecast/update-submission.R:66]: 😆

**Pull request merged**